frontend/packages/dev-console/src/components/pipelines/resource-overview/DynamicResourceLinkList.tsx

```diff
@@ -28,11 +29,22 @@ const DynamicResourceLinkList: React.FC<DynamicResourceLinkListProps> = ({
       <dl>
         <dt>{title}</dt>
         <dd>
-          {links.map(({ name, model }) => {
+          {links.map(({ name, model, displayName = '' }) => {
             const kind = referenceForModel(model);
+            let linkName = name;
+            if (displayName.length > 0 && name !== displayName) {
```

Comment:
nit:
```suggestion
            if (!!displayName && name !== displayName) {
```

Reply:
I have set a default value for displayName `''`  and checking the length explicitly rather implicit type coercion.